### PR TITLE
CNI docs

### PR DESCRIPTION
**Pull request by ogusarenko**

### CODE REVIEW

File: content/docs/concepts/cni.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    enabled: false
+    deployWithOperator: false
+  - provider: custom
+    enabled: false
+    deployWithOperator: false
+```
+
+## Network configuration
+
+The following table includes the details on all of the configurable `network` fields.
```
Comment (MagdaDziadosz) [inline content/docs/concepts/cni.md:79]:
```suggestion
The following table includes details on all of the configurable `network` fields.
```

File: content/docs/concepts/cni.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+| Field | Description | Valid values |  Default |
+|-------|-------------|--------------|----------|
+| `serviceCIDR` | Sets the IPv4 range of IP addresses for services in a Kubernetes cluster. | Valid IPv4 CIDR | `10.96.0.0/16` |
+| `nodePortRange` | Sets the allowed port range for Kubernetes services of the NodePort type. | Valid port range | `32768-35535` |
+| `providers` | Sets the provider for the active CNI. | `calico` | `calico` |
+
+## Providers configuration
+
+The following table includes the details on the configurable settings
```
Comment (MagdaDziadosz) [inline content/docs/concepts/cni.md:89]:
```suggestion
The following table includes details on the configurable settings
```

File: content/docs/concepts/cni.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+for the Calico provider.
+
+| Field   | Description  | Valid values        |  Default     |
+|---------|--------------|---------------------|--------------|
+| `enabled` | Sets the name of the external storage provider. AWS is currently the only available option. | `true` | `true` |
+| `clusterCIDRIPv4` | Sets the IP pool in the Kubernetes cluster from which Pods are allocated. | Valid IPv4 CIDR | `192.168.0.0/16` |
+| `ipAutodetectionMethod` | Sets the autodetecting method for the IPv4 address for the host. | Provider specific[^0] | None |
+| `mode` | Sets the ipv4 overlay networking mode. | `ipip`, `vxlan` | `vxlan` |
```
Comment (MagdaDziadosz) [inline content/docs/concepts/cni.md:97]:
```suggestion
| `mode` | Sets the IPv4 overlay networking mode. | `ipip`, `vxlan` | `vxlan` |
```

File: content/docs/concepts/cni.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+| `vxlanVNI` | Sets the VXLAN VNI for the VXLAN mode. | Valid VNI number | `10000` |
+| `CALICO_STARTUP_LOGLEVEL` | Sets the early log level for `calico/node`. | Provider specific[^0] | `DEBUG` |
+| `FELIX_LOGSEVERITYSCREEN` | Sets the log level for `calico/felix`. | Provider specific[^0] | `DEBUG` |
+
+[^0]: For the valid values, consult your provider documentation.
+
+## Limitations
+
+Components using `nodeports` may have their own specific way to specify the port numbers for NodePorts,
+and they may need to be changed at the same time as the `nodePortRange`.
```
Comment (MagdaDziadosz) [inline content/docs/concepts/cni.md:108]:
```suggestion
Components using `nodeports` may have their own specific way of specifying the port numbers for NodePorts,
and these may need to be changed simultaneusly with the `nodePortRange`.
```

File: content/docs/concepts/cni.md [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+  - provider: custom
+    enabled: false
+    deployWithOperator: false
+```
+
+## Network configuration
+
+The following table includes the details on all of the configurable `network` fields.
+
+| Field | Description | Valid values |  Default |
```
Comment (MagdaDziadosz) [inline content/docs/concepts/cni.md:81]: Too many uses of “valid”, suggesting changing the header to Values, and consider removing valid from the table values as well.

File: content/docs/concepts/cni.md [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+| `serviceCIDR` | Sets the IPv4 range of IP addresses for services in a Kubernetes cluster. | Valid IPv4 CIDR | `10.96.0.0/16` |
+| `nodePortRange` | Sets the allowed port range for Kubernetes services of the NodePort type. | Valid port range | `32768-35535` |
+| `providers` | Sets the provider for the active CNI. | `calico` | `calico` |
+
+## Providers configuration
+
+The following table includes the details on the configurable settings
+for the Calico provider.
+
+| Field   | Description  | Valid values        |  Default     |
```
Comment (MagdaDziadosz) [inline content/docs/concepts/cni.md:92]: Same comment as in the table above.

**Review by MagdaDziadosz — CHANGES_REQUESTED** [review]

File: content/docs/concepts/cni.md (hunk 4 above)
Comment (vikramhh) [reply content/docs/concepts/cni.md:108]: @MagdaDziadosz - spellings of simultaneously

**Review by vikramhh — APPROVED** [review]
LGTM - I am good either way with https://github.com/Mirantis/mke-docs/pull/52#discussion_r1690893236

**Review by MagdaDziadosz — APPROVED** [review]

**Pull request merged**